src/Model/Storage/ISelectableStorage.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2021, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Model\Storage;
+
+/**
+ * Interface for selectable storage backends
```

Comment:
Can you please extend on what "selectable" means in this context?

Reply:
Maybe you have a better naming for this interface: 

It's a storage backend, which an admin can select for "user data" and have the CRUD (create update delete) pattern. The `IStorage` is a much simpler interface which allows just to retrieve data from a storage backend.

Reply:
Why not `Writable`?

Reply:
sounds better ^^